aws_config_policies/aws_config_global_resources.yml

```diff
@@ -17,18 +17,56 @@ Runbook: >
 Reference: https://amzn.to/2MO8xXM
 Tests:
   -
-    Name: Global Resources Are Enabled
-    ExpectedResult: true
-    Resource:
-      {
-        "GlobalRecorderCount": 4,
-        "GlobalResourcesEnabled": true
-      }
-  -
-    Name: Global Resources Are Not Enabled
+    Name: No Global Recoders Present
     ExpectedResult: false
     Resource:
       {
-        "GlobalRecorderCount": 4,
-        "GlobalResourcesEnabled": false
+        "TimeCreated": null,
+        "AccountId": "123456789012",
+        "ResourceId": "123456789012::AWS.Config.Recorder.Meta",
+        "ResourceType": "AWS.Config.Recorder.Meta",
+        "Tags": null,
+        "Region": "global",
+        "Recorders": null,
+        "Name": "AWS.Config.Recorder.Meta",
+        "GlobalRecorderCount": 0
       }
+#  -
+#    Name: __REPLACE_ACCOUNT_ID_TO_SUPPORT_TESTING__ Global Resources Are Enabled
```

Comment:
This name feels sketchy

Reply:
Is this to alert someone that to uncomment they should replace the account id? seems reasonable to me

Reply:
yeah, exactly